go/tendermint/apps/beacon/state/state.go

```diff
@@ -15,6 +16,8 @@ var (
 	//
 	// Value is raw random beacon.
 	beaconKeyFmt = keyformat.New(0x40)
+	// genesisKeyFmt is the key format used for genesis.
+	genesisKeyFmt = keyformat.New(0x41)
```

Comment:
Why do we need to store this in ABCI state, isn't the genesis document always available anyway?

Reply:
I thought we wanted it to be consensus-backed, so that malicious nodes can't modify it on dump/restore.  The beacon's genesis data is basically just a single bool, so it shouldn't take much space.